### PR TITLE
4.3 to 4.4 UAV Tech Presets Update - NEW

### PR DESCRIPTION
**Pull request by spatzengr**
Resubmitting without index update.  Seems like there is items in other presets (by others) to be possibly addressed and I didn't want to hold up this PR because of those items.

Updated my 7 presets coming from 4.3 to 4.4. PID Settings are the same. Mostly changed the header information to be more user-friendly. I did add filter options. Once merged I have 4 new presets to add for which I will create a separate PR for each.

### CODE REVIEW

File: presets/4.4/tune/uav_tech/UAV_tech_5in_Freestyle_575-650.txt [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+set simplified_master_multiplier = 135
+
+set anti_gravity_gain = 80
+set pidsum_limit = 1000
+set pidsum_limit_yaw = 1000
+
+#$ OPTION_GROUP BEGIN: Choose ONE Filter option (+ RPM filter if desired)
+    #$ OPTION BEGIN (UNCHECKED): low Build Quality
+    # -- ADDER: For HIGH gyro vibration builds --
+    set simplified_gyro_filter = ON
```
Comment (limonspb) [inline presets/4.4/tune/uav_tech/UAV_tech_5in_Freestyle_575-650.txt:64]: internal code for options better be also with the extra identation of 4 spaces (8 spaces total if inside option group)

File: presets/4.4/tune/uav_tech/UAV_tech_5in_Freestyle_575-650.txt [hunk 2]
```diff
@@ -0,0 +1,129 @@
+#$ TITLE: UAV Tech - 5" Freestyle withOUT GoPro (575-650g AUW)
+#$ FIRMWARE_VERSION: 4.4
+#$ CATEGORY: TUNE
+#$ STATUS: COMMUNITY
```
Comment (limonspb) [inline presets/4.4/tune/uav_tech/UAV_tech_5in_Freestyle_575-650.txt:4]: so new presets file should be marked as EXPERIMENTAL for safety.
Exceptions are some small trivial presets.

File: presets/4.4/tune/uav_tech/UAV_tech_6-7in.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#$ TITLE: UAV Tech - 6-7in (~700g AUW)
+#$ TITLE: UAV Tech - 6"/7" (750-1000g AUW)
 #$ FIRMWARE_VERSION: 4.4
 #$ CATEGORY: TUNE
 #$ STATUS: COMMUNITY
```
Comment (limonspb) [inline presets/4.4/tune/uav_tech/UAV_tech_6-7in.txt:4]: this has lots of changes, better change to community, at least till a few people get to test it as it is now

File: presets/4.4/tune/uav_tech/UAV_tech_6-7in.txt [hunk 4]
```diff
@@ -67,23 +59,75 @@ set simplified_i_gain = 65
...
+set iterm_relax_cutoff = 10
+set anti_gravity_gain = 90
 set pidsum_limit = 1000
 set pidsum_limit_yaw = 1000
 
-simplified_tuning apply
+#$ OPTION_GROUP BEGIN: Choose ONE Filter option (+ RPM filter if desired)
+    #$ OPTION BEGIN (UNCHECKED): low Build Quality
+    # -- ADDER: For HIGH gyro vibration builds --
+    set simplified_gyro_filter = ON
```
Comment (limonspb) [inline presets/4.4/tune/uav_tech/UAV_tech_6-7in.txt:71]: indentation:
```
option group
    option
         set xxx = yyy
    end option
option group end
```

File: presets/4.4/tune/uav_tech/UAV_tech_8-9in.txt [hunk 5]
```diff
@@ -1,44 +1,55 @@
-#$ TITLE: UAV Tech - 8-9in (~750g AUW)
+#$ TITLE: UAV Tech - 8"/9" (1000-1400 AUW)
 #$ FIRMWARE_VERSION: 4.4
 #$ CATEGORY: TUNE
-#$ STATUS: EXPERIMENTAL
+#$ STATUS: COMMUNITY
```
Comment (limonspb) [inline presets/4.4/tune/uav_tech/UAV_tech_8-9in.txt:4]: same for this, its a big change, better make it experimental for some time.

File: presets/4.4/tune/uav_tech/UAV_tech_Cinelifter_8-9in.txt [hunk 6]
```diff
@@ -0,0 +1,137 @@
+#$ TITLE: UAV Tech - Cinelifter (8"/9" >2,000g AUW)
+#$ FIRMWARE_VERSION: 4.4
+#$ CATEGORY: TUNE
+#$ STATUS: COMMUNITY
```
Comment (limonspb) [inline presets/4.4/tune/uav_tech/UAV_tech_Cinelifter_8-9in.txt:4]: experimental

File: presets/4.4/tune/uav_tech/UAV_tech_Micro.txt [hunk 7]
```diff
@@ -0,0 +1,132 @@
+#$ TITLE: UAV Tech - Micro (2in to 4in)
+#$ FIRMWARE_VERSION: 4.4
+#$ CATEGORY: TUNE
+#$ STATUS: COMMUNITY
```
Comment (limonspb) [inline presets/4.4/tune/uav_tech/UAV_tech_Micro.txt:4]: experimental

**Review by limonspb — CHANGES_REQUESTED** [review]
i've added a few minor comments.
My request would be to separate big preset changes in separate PRs.

Its ok to have small changes in multiple presets. But this is like a huge change for many presets that are independent. Way easier to fix and merge one preset, and then based on the previous experience submit a new PR for a different preset.

Also easier on reviewers. A single file change can be reviewed more carefully

File: presets/4.4/tune/uav_tech/UAV_tech_5in_Freestyle_575-650.txt (hunk 2 above)
Comment (spatzengr) [reply presets/4.4/tune/uav_tech/UAV_tech_5in_Freestyle_575-650.txt:4]: i just renamed the file for the 5" preset that has been in use for a while.

File: presets/4.4/tune/uav_tech/UAV_tech_6-7in.txt (hunk 3 above)
Comment (spatzengr) [reply presets/4.4/tune/uav_tech/UAV_tech_6-7in.txt:4]: just a file rename and change of title

**Pull request closed without merging**